inst/include/Stats.h

```diff
@@ -756,6 +756,133 @@ class NodeMatch : public BaseStat< Engine > {
 typedef Stat<Directed, NodeMatch<Directed> > DirectedNodeMatch;
 typedef Stat<Undirected, NodeMatch<Undirected> > UndirectedNodeMatch;
 
+/*!
+ * Adds a statistic the absolute value of the difference between the values
+ * of a continuous nodal covariate
+ */
+template<class Engine>
```

Comment:
Documentation for the terms should be added to the R level statistic documentation.